packages/websocket-plugin/src/symbols.ts

```diff
@@ -94,3 +94,13 @@ export class SendWebSocketMessage {
   }
   constructor(public payload: any) {}
 }
+
+/**
+ * This error is thrown where there is no `type` (or custom `typeKey`) property
+ * on the message that came from the server side socket
+ */
+export class TypeKeyPropertyMissing extends Error {
```

Comment:
Wait, this is now publically exposed because all symbols are exported to the public api